assets/js/translator.js

```diff
@@ -1111,14 +1114,71 @@ function setRecentDstLangs(langs /*: string[] */) {
     return langs;
 }
 
+function dictionaryLookup() {
+    if(modeEnabled('lookup')) {
+        if(currentLookupRequest) {
+            currentLookupRequest.abort();
+        }
+
+        $('#dictionaryLookupResult').empty();
+        var wordToLookup = $('#originalText').val().trim().split(' '); // eslint-disable-line newline-per-chained-call
+        if(wordToLookup.length <= 3 && wordToLookup[0] !== '') {
+            currentLookupRequest = callApy({
+                data: {
+                    'langpair': curSrcLang + '|' + curDstLang,
+                    'q': $('#originalText').val()
+                },
+                success: handleDictionaryLookupSuccessResponse,
+                error: handleDictionaryLookupErrorResponse,
+                complete: function () {
+                    ajaxComplete();
+                    currentLookupRequest = undefined;
+                }
+            }, '/dictionaryLookup');
```

Comment:
I think @sushain97 said this could be an addition to the `/translate` endpoint? I'm not sure which is better though.

Reply:
Yep. With a `?dictionary=true` or something.